adrs/0153-checkout-v2.md

```diff
@@ -43,12 +43,18 @@ We want to take this opportunity to make behavioral changes, from v1. This docum
   fetch-depth:
     description: 'Number of commits to fetch. 0 indicates all history.'
     default: 1
+  fetch-refs:
```

Comment:
Opted to update existing ADR rather than create a new document. This single file gives a holistic picture.